services/ui-src/src/utils/reports/reports.ts

```diff
@@ -93,3 +93,30 @@ export const compileValidationJsonFromRoutes = (
   });
   return validationSchema;
 };
+
+/*
+ * saving this method here in case we need to programmatically
+ * create a list of fieldIds in the future or want to store
+ * them on the report object at some point. not currently called
+ * anywhere in the app as of 2022.09.23
+ */
```

Comment:
🤷 I can't think of a use case where we'd map them this way on the code side, though I see how it's useful for BOs and downstream

Reply:
Yeah, the code is really useful but David didn't really have an opinion about this being added in the db when I asked him on Friday.

Reply:
Yeah I don't think we'd ever need it on the code side, but it's something that could be referenced if needed if it was in the database. I'm leaning toward not adding it to the db record since it's not really necessary code-wise. We can always re-run it in the future if needed.